marlowe-dashboard-client/src/Capability/Marlowe.purs

```diff
@@ -292,6 +294,9 @@ instance monadMarloweAppM :: ManageMarlowe AppM where
             WithMarloweContracts -> Contract.invokeEndpoint MarloweApp marloweAppId "apply-inputs" (marloweParams /\ Just interval /\ inputs)
       LocalStorage -> do
         existingContracts <- getContracts
+        -- When we emulate these calls we add a 500ms delay so we give time to the submit button
+        -- to show a loading indicator (we'll remove this once the PAB is connected)
+        liftAff $ delay $ Milliseconds 500.0
```

Comment:
The PAB will send the HTTP response very quickly, so we might want to delay that a little bit too... But we can investigate that when it's all hooked up.